tests/test_tracker.py

```diff
@@ -133,3 +132,41 @@ def test_get_extracted_slots_with_no_active_loop():
         "some_other": "some_value2",
         "my_slot": "some_value",
     }
+
+
+def test_get_intent_if_not_fallback():
+    tracker = get_tracker([])
+    assert not tracker.get_intent_if_not_fallback()
+
+    tracker = get_tracker([user_uttered("hello", 0.8)])
+    assert not tracker.get_intent_if_not_fallback()
+
+    tracker = get_tracker(
+        [
+            UserUttered(
+                text="Some Fallback",
+                parse_data={
+                    "intent": {"name": "nlu_fallback", "confidence": 0.9},
+                    "intent_ranking": [{"name": "nlu_fallback", "confidence": 0.9}],
+                },
+            )
+        ]
+    )
+    assert not tracker.get_intent_if_not_fallback()
```

Comment:
optional: I'm personally favoring to test one use case per test function. Hence, I'd either use `pytest.mark.parametrize` for the first two `asserts` and a separate test function for the third `assert` or using `pytest.mark.parametrize` for all three and add a comment about the use case to each input (e.g. `test with empty tracker`, `no fallback after last message`, `last user message triggered fallback` )